app/jiraapi.py

```diff
@@ -159,25 +159,59 @@ def create_domain_approval_task(self, assessment):
             issuetype_name = INITIAL_ASSESSMENT_ISSUE_TYPE
         else:
             summary = 'Profile edit: {} (#{})'.format(application.data.get('name'), application.supplier_code)
-            description = "+*This is a profile edit*+\n\n" \
-                          "Please evaluate the changes made by the seller and ensure they meet the " \
-                          "[assessment guidelines|" \
-                          "https://govausites.atlassian.net/wiki/display/DM/Initial+Assessment+Checklist]. \n" \
-                          "Changes will be summarised at the top of the seller profile.\n\n" \
-                          "Seller profile link: [%s]\n\n" \
-                          "---\n\n" \
-                          "A snapshot of the application is attached." \
-                          "" % (current_app.config['ADMIN_ADDRESS'] +
-                                "/admin/applications/preview/{}".format(application.id))
+            description = (
+                "+*This is a profile edit*+\n\n"
+                "Please evaluate the changes made by the seller and ensure they meet the "
+                "[assessment guidelines|"
+                "https://govausites.atlassian.net/wiki/display/DM/Initial+Assessment+Checklist]. \n"
+                "Changes will be summarised at the top of the seller profile.\n\n"
+                "Seller profile link: [%s]\n\n"
+                "---\n\n"
+                "A snapshot of the application is attached."
+                "" % (current_app.config['ADMIN_ADDRESS'] + "/admin/applications/preview/{}".format(application.id))
+            )
             issuetype_name = SUBSEQUENT_ASSESSMENT_ISSUE_TYPE
+
+        pricing = application.data.get('pricing', None)
+        domain_price_min = 1
+        domain_price_max = 2
+        competitive = False
+        description += "---\n\n"
+        if pricing is not None:
+            for k, v in pricing.iteritems():
+                max_price = v.get('maxPrice')
+                domain = next(d for d in domains if d.name == k)
+                domain_price_min = domain.price_minimum
+                domain_price_max = domain.price_maximum
+                competitive = False
+                if (max_price >= domain_price_min and max_price <= domain_price_max):
+                    competitive = True
+
+                price_description = (
+                    "Price threshold for *{domain_name}*:\n"
+                    "Seller Price: {max_price}\n"
+                    "Minimum Price: {domain_price_min}\n"
+                    "Maximum Price: {domain_price_max}\n"
+                ).format(
+                    max_price=max_price,
+                    domain_name=domain.name,
+                    domain_price_min=domain_price_min,
+                    domain_price_max=domain_price_max
+                )
+                description += price_description
+                if competitive:
```

Comment:
ditch the bool and copy your `if` to this line?

Reply:
removed the extra if condition, but the competitive boolean is needed for later on